test/Test/Database/LSMTree/Internal.hs

```diff
@@ -173,17 +173,16 @@ prop_interimOpenTable dat = ioProperty $
           let snap = fromMaybe (error "invalid name") $ mkSnapshotName "snap"
           numRunsSnapped <- snapshot const snap "someLabel" th
           th' <- open sesh "someLabel" configNoOverride snap
-          lhs <- lookups const ks th id
-          rhs <- lookups const ks th' id
+          lhs <- lookups const ks th
+          rhs <- lookups const ks th'
           close th
           close th'
           -- TODO: checking lookups is a simple check, but we could have stronger
           -- guarantee. For example, we might check that the internal structures
           -- match.
           --
-          -- TODO: remove opaqueifyBlobs once blobs are implemented.
           pure $ tabulate "Number of runs snapshotted" [show numRunsSnapped]
-              $ (Test.opaqueifyBlobs lhs === Test.opaqueifyBlobs rhs)
+               $ lhs === rhs
   where
```

Comment:
Address the TODO below that is on `runDataWithoutBlobs`

Reply:
There is also one in `prop_roundtripCursor`

Reply:
There is also a TODO regarding blob references in `Test.Database.LSMTree.Internal.RunReaders.arbitraryWithVars`

Reply:
And `Test.Database.LSMTree.Normal.Examples.tests`

Reply:
I grepped for TODOs mentioning blobs and I listed the ones we still have to remove above. It's not really all in scope for this PR, but they are probably small enough changes that you can make them in this PR.

Reply:
Thanks for the audit. I'll have a look at the others.

Reply:
Looks good!